src/template-tags/tickets.php

```diff
@@ -181,31 +181,31 @@ function tribe_events_count_available_tickets( $event = null ) {
 if ( ! function_exists( 'tribe_tickets_buy_button' ) ) {
 
 	/**
-	 * Echos Remaining Ticket Count and Purchase Buttons for an Event
+	 * Echo remaining ticket count and purchase/rsvp buttons for a post.
 	 *
 	 * @since  4.5
+	 * @since  TBD Now displays for posts having only RSVPs.
 	 *
 	 * @param bool $echo Whether or not we should print
 	 *
 	 * @return string
 	 */
 	function tribe_tickets_buy_button( $echo = true ) {
-		$event_id = get_the_ID();
+		$event = get_post();
```

Comment:
Do we need the full object in this function? If not, we can use `get_the_ID()` and bail if `false === $event_id`

Reply:
OK.